packages/core/src/controllers/verify.ts

```diff
@@ -62,35 +62,21 @@ export class Verify extends IVerify {
 
   public register: IVerify["register"] = async (params) => {
     if (!isBrowser()) return;
+    const origin = window.location.origin;
     const { id, decryptedId } = params;
-    const url = `${this.verifyUrlV2}/attestation?projectId=${this.core.projectId}`;
-    let src = "";
-    try {
-      const response = await fetch(url, {
-        method: "POST",
-        body: JSON.stringify({ id, decryptedId }),
-      });
-      const { srcdoc } = await response.json();
-      src = srcdoc;
-      this.logger.debug("srcdoc fetched", src);
-    } catch (e) {
-      this.logger.warn(e);
-      return;
-    }
+    const src = `${this.verifyUrlV3}/attestation?projectId=${this.core.projectId}&origin=${origin}&id=${id}&decryptedId=${decryptedId}`;
     try {
       const document = getDocument() as Document;
-      const abortTimeout = this.startAbortTimer(ONE_SECOND * 3);
+      const abortTimeout = this.startAbortTimer(ONE_SECOND * 5);
```

Comment:
```suggestion
      const abortTimeout = this.startAbortTimer(ONE_SECOND * 5);
```
this would block the publish for 5s in worst case scenario where verify iframe doesn't load, just noting it

Reply:
I almost thing it should be higher, especially in bad network conditions. Perhaps we should add an onerror handler to the iframe?

Reply:
Added `onerror` handler in 47e3471d815aab083bd81e0ea4c75a0500e776ff